.github/workflows/plugin.yaml

```diff
@@ -16,6 +16,12 @@ jobs:
         pex-tool:
           - default
           - in-repo
+        exclude:
+          # Disabled until a please_pex release works with Python >= 3.12:
+          - python-version: '3.12'
+            pex-tool: default
+          - python-version: '3.13'
+            pex-tool: default
```

Comment:
We'll need to exclude these from testing temporarily until the workflow uploads a version of please_pex that supports Python >= 3.12, then they can be re-added.